api/db-migrations/000013_add_default_traffic_rule.down.sql

```diff
@@ -0,0 +1,50 @@
+WITH dangling_routes AS (
+    SELECT
+        id
+        , default_traffic_rule -> 'routes' AS routes
+    FROM
+        router_versions
+    WHERE default_traffic_rule -> 'routes' IS NOT NULL
+),
+cleaned_router_versions AS (
+    SELECT id, CASE WHEN traffic_rules = 'null' THEN '[]'::jsonb ELSE traffic_rules END AS traffic_rules
+    FROM router_versions
+),
+exploded_traffic_rules AS (
+    SELECT
+        id
+        , position
+        , elem
+    FROM cleaned_router_versions, jsonb_array_elements(traffic_rules) WITH ordinality arr(elem, position)
+    WHERE id IN (SELECT id FROM dangling_routes)
```

Comment:
Added a filter to only alter traffic_rules if there are dangling routes.

Reply:
This makes sense for the down query. 👍🏽